### PR TITLE
Add hhkb-like-arrows-and-page-control

### PR DESCRIPTION
**Pull request by gun0317**
Fn + [;/' to arrows
Fn + l to page up
Fn + . to page down

### CODE REVIEW

File: public/groups.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
       "name": "Modifier Keys",
       "id": "modifier-keys",
       "files": [
+	{
+	  "path": "json/hhkb-like-arrows-and-page-control.json"
+	},
```
Comment (MuhammedZakir) [inline public/groups.json:9]: This should be inside "emulation mode" instead of "modifier keys".

**Review by MuhammedZakir — CHANGES_REQUESTED** [review]

File: public/groups.json (hunk 1 above)
Comment (gun0317) [reply public/groups.json]: Thanks

File: public/groups.json [hunk 2]
```diff
@@ -125,6 +125,9 @@
       "name": "Emulation Modes",
       "id": "emulation-modes",
       "files": [
+	{
+	  "path": "json/hhkb-like-arrows-and-page-control.json"
+	},
```
Comment (MuhammedZakir) [inline public/groups.json:130]:
```suggestion
        {
          "path": "json/hhkb-like-arrows-and-page-control.json"
        },
```

**Comment by MuhammedZakir** [comment]
Merged. :-)

**Pull request merged**